src/main/java/com/salesforce/mce/awesolog/aws/S3ClientConstructor.java

```diff
@@ -32,14 +32,17 @@ public AwsS3Client(
         this.s3Region = s3Region;
     }
 
-    public S3Client build() {
+    public S3Client construct() {
 
         S3Client s3Client;
+        StsClient stsClient = null;
+
         S3ClientBuilder s3ClientBuilder;
         StsClientBuilder stsClientBuilder;
 
         StaticCredentialsProvider staticCredentialsProvider = null;
-        StsClient stsClient = null;
+
+        final String AssumeRoleSessionName = "AwesologSession";
```

Comment:
let's use `ASSUME_ROLE_SESSION_NAME` for java constants

Reply:
updated